suse2013/xhtml/xref.xsl

```diff
@@ -139,11 +139,13 @@
   </xsl:call-template>
 
  <!--
-    The xref resolution is a bit tricky. We need to distinguish 3 cases:
+    The xref resolution is a bit tricky. We need to distinguish these cases:
 
     1. if $rootid = '' -> use original code
     2. if $rootid != '' and rootid points to the root node -> use original code
     3. if (1) or (2) does not apply -> reference into another book
+    4. if we point to another book AND we have an @xrefstyle, output the
+       text but don't make it clickable
```

Comment:
Suggestion that clarifies this a bit more:
```
4. if we point to another book AND we have an @xrefstyle -> output the
       same text content as the original xref template but remove the link
```

Reply:
It's integrated in commit 41888ae.